doc/general/reference-tables/reference-tables.md

```diff
@@ -131,10 +126,10 @@ For definitions regarding national laws, please refer to the appropriate appendi
 
 | **Value**            | **Description**                              | **Middleware-Version** |
 |----------------------|----------------------------------------------|------------------------|
-| `0x0000000000000000` | **Version information** <br /> <br /> Version information informs which version of the fiskaltrust.Middleware is currently being used |  1.1                    |
-| `0x0000000000000001` | **[fiskaltrust.ActionJournal](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/general/cash-register-integration#fiskaltrustactionjournal)** in internal format <br /> <br /> The fiskaltrust.ActionJournal collects all operational incidents. This can be the date and time of start or failure of the service, as well as any other information related to the fiskaltrust.Middleware and fiskaltrust.SecurityMechanism.| 1.1                    |
-| `0x0000000000000002` | **[ReceiptJournal](https://docs.fiskaltrust.cloud/docs/poscreators/middleware-doc/general/cash-register-integration#fiskaltrustreceiptjournal)** in internal format <br /> <br /> The fiskaltrust.ReceiptJournal is used to record, hash, and chain all requests to the fiskaltrust.Middleware and the resulting responses. The first part of the returned ReceiptIdentification is an upcounting number generated by ReceiptJournal.           | 1.1                    |
-| `0x0000000000000003` | **QueueItemJournal in internal format** <br /> <br /> QueueItemJournal shows every information related to to the fiskaltrust.Middleware and fiskaltrust.SecurityMechanism as wellas all  receipts sent to the fiskaltrust.Middleware and all resulting responses. This is useful for archiving purposes.         | 1.1                    |
+| `0x0000000000000000` | **Version information** </br> </br> Version information informs which version of the fiskaltrust.Middleware is currently being used |  1.1                    |
```

Comment:
Please don't use `</br`, as this will break the Docusaurus build :D only `<br />` is valid MDX.

Also applies for the changes below

Reply:
The modifications have been made, in the commit: b08f5afccc8534edb4c94d9d37d065f0990c9816. :)